### ISSUE TITLE
TENANT_LIMIT_SET_CALLS does't reduce SET search_path calls

### ISSUE THREAD

**Issue by webjunkie** [tomturner/django-tenants#155]
Even if I activate `TENANT_LIMIT_SET_CALLS`, I see in my log the same amount of `SET` statements.

How to reproduce:

Activate `TENANT_LIMIT_SET_CALLS` and the DB logs:

```python
        'django.db.backends': {
            'level': 'DEBUG',
            'handlers': ['console'],
        }
```

It seems to boil down to the fact that the `DatabaseWrapper` compares cursors, but my debugger is telling me I get a new cursor before each and every request anyways, thus the limiting of set calls is never effective. Is that unique to my setup?

This is the line:

    if (not get_limit_set_calls()) or not self.search_path_set or self._previous_cursor != cursor:

It might have to do with #115 and it's actually a little sad that this made it in without any tests whatsoever.

**Comment by tomturner** [comment]
Merged the PR that fixes this

**Comment by webjunkie** [comment]
For future references, this is closed with #157
